package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@capacitor-community/fcm",
-  "version": "2.0.2",
+  "name": "@textable-app/fcm",
```

Comment:
@davidhulick  Why did you rename plugin name??

Reply:
@rdlabo My apologies. I created a fork of this repo for use in my own project and committed to master so that I could publish to our own npm namespace. Unfortunately, I don't believe I can change the target branch of a Pull Request. Should I close this Pull Request and create a new one from [davidhulick:feature/capacitor4](https://github.com/davidhulick/fcm/tree/feature/capacitor4) which contains the changes that I intend to merge into @community-capacitor/fcm ?